### ISSUE TITLE
JSON field isnull filter

### ISSUE THREAD

**Issue by vekkuli** [collerek/ormar#529]
Using a nullable JSON field and filtering with isnull produces unexpected results. Is the JSON intended to be treated differently when it comes to nullness?

**To reproduce and expected behavior:**

```
import asyncio
import databases
import ormar
import sqlalchemy

DATABASE_URL = "sqlite:///db.sqlite"
database = databases.Database(DATABASE_URL)
metadata = sqlalchemy.MetaData()

class Author(ormar.Model):
    class Meta(ormar.ModelMeta):
        metadata = metadata
        database = database
        tablename = "authors"
    id = ormar.Integer(primary_key=True)
    text_field = ormar.Text(nullable=True)
    json_field = ormar.JSON(nullable=True)

engine = sqlalchemy.create_engine(DATABASE_URL)
metadata.drop_all(engine)
metadata.create_all(engine)

async def test():
    async with database:
        author = await Author.objects.create()
        assert author.json_field is None
        non_null_text_fields = await Author.objects.all(text_field__isnull=False)
        assert len(non_null_text_fields) == 0
        non_null_json_fields = await Author.objects.all(json_field__isnull=False)
        assert len(non_null_json_fields) == 0 # Fails

asyncio.run(test())
```

**Comment by ponytailer** [comment]
Maybe is : `Author.json_field.isnull(None)`

**Comment by collerek** [comment]
Fixed in 0.10.25 when you set field to nullable None value is treated as null in sqlalchemy
